pontoon/base/templates/project_contributors.html

```diff
@@ -0,0 +1,16 @@
+{% extends "project.html" %}
+{% import 'widgets/contributors.html' as Contributors %}
+
+{% block title %}{{ super() }} - contributors {% endblock %}
```

Comment:
Instead of dash and lowercase (- contributors), let's keep the pattern and use: &middot; Contributors.